wp-content/plugins/wporg-learn/js/course-completion-meta/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+import { PanelRow, TextControl } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { PluginDocumentSettingPanel } from '@wordpress/edit-post';
+import { __ } from '@wordpress/i18n';
+import { registerPlugin } from '@wordpress/plugins';
+
+const CourseCompletionMeta = () => {
+	const postMetaData = useSelect( ( select ) => select( 'core/editor' ).getEditedPostAttribute( 'meta' ) || {} );
+	const { editPost } = useDispatch( 'core/editor' );
+
+	const message = postMetaData?._course_completion_success_message || '';
+	const link = postMetaData?._course_completion_survey_link || '';
+
+	return (
+		<PluginDocumentSettingPanel title={ __( 'Course Completion Settings', 'wporg-learn' ) }>
+			<PanelRow>
+				<p>{ __( 'If the fields are left blank, the default values will be applied.', 'wporg-learn' ) }</p>
+			</PanelRow>
+			<PanelRow>
+				<TextControl
```

Comment:
Can we show a placeholder value here 'Congratulations on completing this course!'?

Reply:
![image](https://github.com/user-attachments/assets/4906a95f-0fc9-40a4-b25b-494475c7487c)
The placeholder is getting cut off though. What do you think about shortening it to something like: 'Course completed!' or just leaving it blank like the other fields?

Reply:
Personally I don't have a problem with it being cut off; I think it still provides an example of what is expected. If we did want to shorten it I'd just use `Congratulations!`, as it's still close to the default.

Reply:
> I think it still provides an example of what is expected.

This makes sense to me. 16df720